aten/src/ATen/native/cudnn/RNN.cpp

```diff
@@ -502,26 +616,76 @@ namespace {
                 lin_layer_mat_desc.mut_desc(),
                 &matrix_pointer
                 ));
+#else
+          void *unused_pointer;
```

Comment:
Mind assigning it to null?
```suggestion
          void *unused_pointer = nullptr;
```